source/PlayerInfoPanel.h

```diff
@@ -61,6 +62,8 @@ class PlayerInfoPanel : public Panel {
 	void DrawPlayer(const Rectangle &bounds);
 	void DrawFleet(const Rectangle &bounds);
 
+	void DrawTooltip(const std::string &text, const Point &hoverPoint);
```

Comment:
This should probably be in the anonymous namespace in PlayerInfoPanel.cpp; I don't it being called from anywhere that doesn't have access to that scope.

Reply:
OK.